### ISSUE TITLE
Question of formula 8

### ISSUE THREAD

**Issue by lvwuyunlifan** [jeong-tae/RACNN-pytorch#13]
hi, @jeong-tae : 
    I looked at your code and looked at the model and original paper of the source racnn-caffe. I have a question about the final loss function. The original racnn-caffe model finally concat **pool5, pool5_A, and pool5_A_A**(pow1, pow2, pow3), and then calculates accuracy1+2+3 through the fc layer. Figure:
![37](https://user-images.githubusercontent.com/27217235/48937433-00cebd00-ef49-11e8-939b-00f266e62730.png)

I read your code and the original paper about this Lcls. You calculate each Lcls1, Lcls2, Lcls3, and then add them up. Figure:
![d](https://user-images.githubusercontent.com/27217235/48937435-05937100-ef49-11e8-94ee-c4be1f43071c.png)

I don't think they are a meaning. If I want to use scale1+2+3 like the source racnn-caffe, how I  do it ?

**Comment by jeong-tae** [comment]
oh, yes. you are right. If you see the code in RACNN.py. those loss accuracy1+2+3 and accuracy1+2 are commented out. This is because to check each loss performance to reproduce exactly.

If you want to calculate original feature fusion loss, remove comment '#' and change loss part.

I will think about this two separated setting to train. Thx!

**Comment by lvwuyunlifan** [comment]
oh, yeah. I know what to do, thank you.

**Comment by lvwuyunlifan** [comment]
Oh, I know what to do. Thank you~! I didn't notice the commented part before, but I wrote the concat part code myself. I have two questions here. Do I have to add the pow function, configure sacle, shift, power like the source caffe, and what effect does the pow function have on the fc layer and accuracy?  Another problem is, if I want to use accuracy1+2+3(loss1+2+3) or accuracy1+2, L(X) is loss1+2+3 + Lrank (lrank1 + lrank2)?

**Comment by lmy418lmy** [comment]
hi   @lvwuyunlifan ，
could you send me the original source about racnn-caffe？

**Comment by rechealzrq** [comment]
> 嗨，@ jeong-tae：
> 我看过您的代码，看了看racnn-caffe的模型和原始论文。我对最终损失函数有疑问。最初的racnn-caffe模型最终连接了**pool5，pool5_A和pool5_A_A**（pow1，pow2，pow3），然后通过fc层计算准确性1 + 2 + 3。数字：
> ![37](https://user-images.githubusercontent.com/27217235/48937433-00cebd00-ef49-11e8-939b-00f266e62730.png)
> 
> 我阅读了您的代码和有关此Lcl的原始论文。您计算每个Lcls1，Lcls2，Lcls3，然后将它们相加。数字：
> ![d](https://user-images.githubusercontent.com/27217235/48937435-05937100-ef49-11e8-94ee-c4be1f43071c.png)
> 
> 我认为它们不是意思。如果我想像源racnn-caffe一样使用scale1 + 2 + 3，该怎么办？



> 嗨，@ jeong-tae：
> 我看过您的代码，看了看racnn-caffe的模型和原始论文。我对最终损失函数有疑问。最初的racnn-caffe模型最终连接了**pool5，pool5_A和pool5_A_A**（pow1，pow2，pow3），然后通过fc层计算准确性1 + 2 + 3。数字：
> ![37](https://user-images.githubusercontent.com/27217235/48937433-00cebd00-ef49-11e8-939b-00f266e62730.png)
> 
> 我阅读了您的代码和有关此Lcl的原始论文。您计算每个Lcls1，Lcls2，Lcls3，然后将它们相加。数字：
> ![d](https://user-images.githubusercontent.com/27217235/48937435-05937100-ef49-11e8-94ee-c4be1f43071c.png)
> 
> 我认为它们不是意思。如果我想像源racnn-caffe一样使用scale1 + 2 + 3，该怎么办？

Hello, Did you find these two files in the picture?
![FPA_S_~ZYMA(OUF0RE`J(1](https://user-images.githubusercontent.com/43806648/67646136-a06daa00-f967-11e9-9dd5-326177c40453.png)
![FG}D_HZST5RHL{21L5D7M93](https://user-images.githubusercontent.com/43806648/67646144-ac596c00-f967-11e9-9ce3-343bcacb5890.png)
 I can't find these two files in the downloaded resource.
